### ISSUE TITLE
Bookstack

### ISSUE THREAD

**Issue by antwacky** [djjudas21/charts#59]
### Describe the bug a clear and concise description of what the bug is.

Dude, where's bookstack gone??? :'(

### What's your helm version?

version.BuildInfo{Version:"v3.12.3", GitCommit:"3a31588ad33fe3b89af5a2a54ee1d25bfe6eaa5e", GitTreeState:"clean", GoVersion:"go1.20.7"}

### What's your kubectl version?

v1.28.4

### Which chart?

Bookstack

### What's the chart version?

5.2.7

### What happened?

Bookstack has gone :(

### Enter the command that you execute and failing/misfunctioning.

helm install bookstack

**Comment by djjudas21** [comment]
Hey, sorry about this. Bookstack was deprecated at this repo in 9e31d1f8a0940920ea6f7a80603f5ab2e413ce79, then deleted in 9abbaa5c2e72f0b7d20c37591105ef3d1b347efe and then finally purged from the metadata in 1fad1ef61df88665d2ccc077b603fdca0da12b23

I recommend using [gabe565/bookstack](https://artifacthub.io/packages/helm/gabe565/bookstack) which is what I am also using for my deployment of Bookstack.

Long story short, @gabe565 and I both took on a bunch of the charts we were using when [k8s-at-home deprecated their repo](https://github.com/k8s-at-home/charts/issues/1761) at short notice. We later found that we had some duplicates between us, so I dropped a few of the ones I copied.

Hope this helps!

**Comment by antwacky** [comment]
No problem, thanks for pointing me to the other repo!
